#607 Update docs for new users (#608)
slackapi/hubot-slack: docs/index.md

Next, you'll need a token from Slack to authenticate your Hubot. Use one of the following choices:

- **Create a [Classic Slack App]((https://api.slack.com/authentication/migration#classic)) with a Bot User (recommended)**: Slack apps are a container for many capabilities in the Slack platform, and let you access those capabilities in a single place. This is the recommended choice because it allows room for your Hubot to grow. To begin, you just need a Bot User.

    1. Create a new [Classic Slack app](https://api.slack.com/authentication/migration#classic) at the [app management page](https://api.slack.com/apps?new_classic_app=1). Pick a clever name and choose the workspace you want Hubot installed in.
    2. Navigate to the Bot User page and add a bot user. The display name is the name your team will use to mention your Hubot in Slack.
    3. Navigate to the Install App page and install the app into the workspace. Once you've authorized the installation, you'll be taken back to the Install App page, but this time you'll have a **Bot OAuth Access Token**. Copy that value, it will be your Slack token.

>     **⚠️** Although the admin pages encourages you to upgrade your app's OAuth scopes to the latest model, please do not upgrade as long as you use it along with Hubot. The latest permission model doesn't support RTM API, the underlying API of Hubot adapter.

Reviewer: ```suggestion
    **⚠️** Although the admin page encourages you to upgrade your app's OAuth scope, please do not do so. The latest permission model does not support the RTM API, which is the underlying API of the Hubot adapter.
```

@shaydewael I'm taking a stab at this, but my understanding of Hubot is limited so not sure I'm referring to it correctly here.

Author: @misscoded Thanks, this doesn't change the meaning at all and it's far better.

> - **Create a configuration of the Hubot Integration**: The [Hubot Integration](https://my.slack.com/apps/A0F7XDU93-hubot) is an older way to use the Slack platform. It's main advantage is you get more permissions out of the box. For some more-security-sensitive people, this might be a disadvantage. From the above link click Install, choose a username, and finish by clicking Add Hubot Integration. Onthe next page, you'll see an **API Token**. Copy that value, it will be your Slack token.

Reviewer: ```suggestion
- **Create a configuration of the Hubot Integration**: The [Hubot Integration](https://my.slack.com/apps/A0F7XDU93-hubot) is an older way to use the Slack platform. Its main advantage is that you get more permissions out of the box. For those concerned with security, this may be undesirable and should be taken into consideration. 

From the link above, click Install, choose a username, and finish by clicking Add Hubot Integration. On the following page, you'll see an **API Token**. Copy that value, as it will be your Slack token.
```

Feel free to disregard the above. This is the first time I've read this copy and just made some suggestions that made the meaning clearer (at least to me). It's a lengthy paragraph, so not sure if breaking it up into two sections adversely affects the layout.

Author: I'm not the author of the sentences (I just removed line breaks this time) but it looks great to me.
